src/test/java/org/opengis/cite/cdb10/level1/VerifyLightsXmlStructureTests.java

```diff
@@ -0,0 +1,112 @@
+package org.opengis.cite.cdb10.level1;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.opengis.cite.cdb10.CDBStructure.LightsXmlStructureTests;
+import org.opengis.cite.cdb10.TestFixture;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+/**
+ * Created by martin on 2016-09-01.
+ */
+public class VerifyLightsXmlStructureTests extends TestFixture<LightsXmlStructureTests> {
+
+    private Path metadata;
+    private static File duplicatedCodeLightsXmlFile = new File(System.getProperty("user.dir") + "/src/test/java/org/opengis/cite/cdb10/fixtures/invalid/LightsDuplicatedCode.xml");
```

Comment:
Use Paths to join these folders instead of hard coding, or the tests will break when run on a Windows machine.